### PR TITLE
add crud for flats

### PR DESCRIPTION
**Pull request by jowarni**
Flats controller + views ok

### CODE REVIEW

File: app/controllers/flats_controller.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def index
   end
 
   def show
-    @flat = Flat.find(paramas[:id])
```
Comment (douvel) [inline app/controllers/flats_controller.rb:7]: Nice one :wink:

File: app/controllers/flats_controller.rb [hunk 2]
```diff
@@ -14,6 +14,7 @@ def new
   def create
     @flat = Flat.new(flat_params)
     @flat.save
+    redirect_to flats_path
```
Comment (douvel) [inline app/controllers/flats_controller.rb]: You may redirect to show page (idem for update).

File: app/views/flats/edit.html.erb [hunk 3]
```diff
@@ -1,2 +1,41 @@
-<h1>Flats#edit</h1>
-<p>Find me in app/views/flats/edit.html.erb</p>
+<h1>EDIT</h1>
+<p>Edit your flat:</p>
+<%= form_for(@flat) do |t| %>
```
Comment (douvel) [inline app/views/flats/edit.html.erb:3]: You should use `simple_form` (idem for all forms).

File: app/views/flats/edit.html.erb [hunk 4]
```diff
@@ -1,2 +1,41 @@
...
+<li>
+  <%= t.label :type %>
```
Comment (douvel) [inline app/views/flats/edit.html.erb:5]: Should be `kind`

File: app/views/flats/edit.html.erb (hunk 3 above)
Comment (douvel) [inline app/views/flats/edit.html.erb:3]: Why `t? Should be`f`or`form`.

File: app/views/flats/index.html.erb [hunk 5]
```diff
@@ -2,7 +2,9 @@
 <p>Here are all the flats:</p>
 <ul>
 <%@flats.each do |flat|%>
-<li><%=flat.title%> - Price : <%=flat.price%> - City : <%=flat.city%></li><br>
+<li><%=flat.title%> - Price : <%=flat.price%> - City : <%=flat.city%> - <%= link_to("More details", flat_path(flat))%> - <%= link_to("Edit", edit_flat_path(flat))%> - <%= link_to("Delete", flat_path(flat), :method=>:delete)%></li><br>
```
Comment (douvel) [inline app/views/flats/index.html.erb:5]: Old hash syntax spotted!

File: app/views/flats/index.html.erb [hunk 6]
```diff
@@ -2,7 +2,9 @@
...
 <%end%>
 </ul>
 
+<%= link_to("Create a new flat", new_flat_path())%>
```
Comment (douvel) [inline app/views/flats/index.html.erb]: Useless parentheses.

**Comment by douvel** [comment]
Review done.

**Comment by jowarni** [comment]
ok

**Pull request merged**